v2/stacks/cmd/core-ons.yml

```diff
@@ -15,11 +15,57 @@ services:
     extends:
       file: ${PATH_MANIFESTS}/core-ons/zebedee.yml
       service: zebedee
+    environment:
+      ENABLE_JWT_SESSIONS:            "true"
+      ENABLE_PERMISSIONS_AUTH:        "true"
+    depends_on:
+      dp-identity-api:
+        condition: service_healthy
   florence:
     extends:
       file: ${PATH_MANIFESTS}/core-ons/florence.yml
       service: florence
+    environment:
+      ENABLE_NEW_SIGN_IN: "true"
+      ENABLE_DATASET_IMPORT:  "true"
   dp-frontend-dataset-controller:
     extends:
       file: ${PATH_MANIFESTS}/core-ons/dp-frontend-dataset-controller.yml
       service: dp-frontend-dataset-controller
+  dp-permissions-api:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-permissions-api.yml
+      service: dp-permissions-api
+    depends_on:
+      dp-api-router:
+        condition: service_healthy
+  dp-identity-api:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-identity-api.yml
+      service: dp-identity-api
+  dp-download-service:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-download-service.yml
+      service: dp-download-service
+  dp-dataset-exporter:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-dataset-exporter.yml
+      service: dp-dataset-exporter
+    extra_hosts:
+      - "host.docker.internal:host-gateway"          # localhost mapping for linux
+      - "docker.for.mac.host.internal:host-gateway"  # localhost mapping for mac (docker v17.12 to v18.02)
+  dp-dataset-exporter-xlsx:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-dataset-exporter-xlsx.yml
+      service: dp-dataset-exporter-xlsx
+  babbage:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/babbage.yml
+      service: babbage
+    environment:
+      FORMAT_LOGGING: ${FORMAT_LOGGING:-true}
+      ELASTIC_SEARCH_SERVER:    ${ELASTIC_SEARCH_SERVER:-cmdelasticsearch}
```

Comment:
I'm not sure that babbage connects to cmdelasticsearch, but I could be wrong.

Reply:
Babbage needs an elasticsearch endpoint to run, and there is only the cmd elasticsearch in this stack.  Even though it doesn't interact other than a healthcheck it seems to work.

Reply:
that's interesting - maybe a comment to that effect in this file then just to indicate this?

Reply:
I've added a comment to note this